subscription-manager.spec

```diff
@@ -1119,6 +1119,9 @@ scrollkeeper-update -q -o %{_datadir}/omf/%{name} || :
 %endif
 %endif
 
+%pre
+killall rhsmd > /dev/null 2>&1
+
```

Comment:
I try to understand how this change could solve issues described in https://bugzilla.redhat.com/show_bug.cgi?id=1643189. Please, could you give brief explanation? I can imagine that it can help in situation, when Satellite is not used anymore and subscription.rhsm.redhat.com is used again, but it is quite rare case.

Shouldn't be the `rhsmd` killed, when bootstrap is installed and /etc/rhsm/rhsm.conf is updated?

Reply:
@wottop Thanks for explanation. This change is OK and it can help in described case.

We should IMHO implement proper solution using i-notify as it is used by rhsm.service later.

Reply:
BTW: Builds are failing on all SuSE test machines. It seems that suse machines do not like this change in spec file for some reason.